kotlin-analysis-test/src/main/kotlin/org/jetbrains/research/ml/kotlinAnalysis/util/FileTestUtil.kt

```diff
@@ -39,7 +44,7 @@ object FileTestUtil {
         inFormat: TestFileFormat = TestFileFormat("in", Extension.KT, Type.Input),
         outFormat: TestFileFormat? = null
     ): Map<File, File?> {
-        val (files, folders) = File(folder).listFiles().orEmpty().partition { it.isFile }
+        val files = File(folder).listFiles().orEmpty()
```

Comment:
Why you changed it?

Reply:
Because if we want to get projects from given directory we do not need to go threw it recursively(only first level contains their root dirs). And also I thought recursively getting files from given dir is extra. I can move Extension.DIR processing request to separate method or leave as is and mention it (that it is not recursive now) in the function comment.